definitions/private-law/json/CaseField/ListOnNotice/CaseField.json

```diff
@@ -0,0 +1,13 @@
+[
+  {
+    "LiveFrom": "31/10/2022",
+    "CaseTypeID": "PRLAPPS",
+    "ID": "selectedReasonsForListOnNotice",
+    "Label": "Provide reasons why the without notice application should be heard on notice",
+    "FieldType": "MultiSelectList",
+    "HintText": "These reasons will be shared with the applicant",
```

Comment:
Instead of hint text you can add a new label with bold

Reply:
this cant'be done in a new label to appear in middle as the text needs to be in the middle of the heading which is optional to indicate the checkboxes selection ..we can discuss on this